central/views/images/db_response.go

```diff
@@ -2,14 +2,19 @@ package images
 
 import (
 	"github.com/stackrox/rox/central/views/common"
+	"github.com/stackrox/rox/pkg/features"
 )
 
 type imageResponse struct {
 	common.ResourceCountByImageCVESeverity
-	ImageID string `db:"image_sha"`
+	ImageID   string `db:"image_sha"`
+	ImageV2ID string `db:"id"`
```

Comment:
The db tag should be `image_id` since the search field is `Image ID` in the proto definition.